src/act/declaration.rs

```diff
@@ -0,0 +1,52 @@
+use proc_macro2::TokenStream;
```

Comment:
Can we combine Declaration and to_token_stream?

Reply:
ToTokenStream is no longer a trait. It has been broken into three distinct concepts, two of which where able to form traits and the third is not defined tightly enough to make sense as a trait. A declaration is for the declaration of a type, function, or class. A type annotation is for a type when it is being used as a type annotation. And to_token_stream is used for a catch all. Like when we are forming a piece of a declaration (like the record member has a to_token_stream that record calls on each of it's members when making it's own declaration) or the ACT has a to_token_stream that will create the final rust output.